_component_design/tooltip.md

```diff
@@ -11,7 +11,7 @@ resource: true
 last-modified: 2018-08-02
 helix-ui-css: true
 helix-ui-javascript: true
-pagelink: https://rackerlabs.github.io/helix-ui/components/tooltips/
+pagelink: components/tooltip/
 ---
 
 {% include toc.html %}
```

Comment:
(non-blocking, but should be done soon)
"Click Interaction" should be removed. 

Tooltips should not be toggleable for the purposes of interacting with contained elements.

Reply:
yeah -- this needs to be cleaned out.  later.